CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+Thanks for contributing to WordCamp.org! This page will explain all the ways you can contribute.
```

Comment:
`SHOULD` - We can maybe just link to the existing Handbook page, to keep things DRY.